apps/backoffice-v2/src/domains/workflows/hooks/mutations/useCreateWorkflowMutation/useCreateWorkflowMutation.ts

```diff
@@ -0,0 +1,35 @@
+import { createWorkflow } from '@/domains/workflows/fetchers';
+import { AnyObject } from '@ballerine/ui';
+import { useMutation, useQueryClient } from '@tanstack/react-query';
+import toast from 'react-hot-toast';
+
+export interface WorkflowCreationParams {
+  workflowDefinitionId: string;
+  entityData: AnyObject;
+}
+
+export const useCreateWorkflowMutation = () => {
+  const queryClient = useQueryClient();
+
+  return useMutation({
+    mutationFn: ({
+      workflowDefinitionId,
+      context,
+    }: {
+      workflowDefinitionId: string;
+      context: AnyObject;
+    }) =>
+      createWorkflow({
+        workflowDefinitionId: workflowDefinitionId,
+        context,
+      }),
+    onSuccess: () => {
```

Comment:
See if adding the new workflow to the workflows list as an optimistic update is low effort. Can copy existing optimistic logic to do so.

Reply:
Its not needed, queries are invalidated after creation so list is automatically reloaded. If i understood correctly what you ment.

Reply:
The effort is key. If its low it can be nice to see a case immediately added to the UI on the cases list.